### PR TITLE
Avoid imporing code using unstable APIs

### PR DESCRIPTION
**Pull request by wojpawlik**
Works around https://github.com/denoland/deno_std/issues/1015.

I updated `std` while at it.

I don't know how to test this.

### CODE REVIEW

File: packages/common/src/runtimes/DenoRuntime.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // @ts-ignore
-import * as stdFs from "https://deno.land/std@0.89.0/fs/mod.ts";
+import { ensureDir, ensureDirSync } from "https://deno.land/std@0.104.0/fs/expand_glob.ts";
 // @ts-ignore
-import * as stdPath from "https://deno.land/std@0.89.0/path/mod.ts";
+import { expandGlob, expandGlobSync } from "https://deno.land/std@0.104.0/fs/ensure_dir.ts";
```
Comment (dsherret) [inline packages/common/src/runtimes/DenoRuntime.ts]: Oops. The module specifiers are backwards.

**Review by dsherret — COMMENTED** [review]
Thanks for this PR! I've been meaning to do it, but have been lazy.

File: packages/common/src/runtimes/DenoRuntime.ts (hunk 1 above)
Comment (wojpawlik) [reply packages/common/src/runtimes/DenoRuntime.ts]: Fixed.

**Comment by dsherret** [comment]
Thanks @wojpawlik! I'll do a release later.

**Pull request merged**